### ISSUE TITLE
Tokyo

### ISSUE THREAD

**Issue by KBebonan** [hudayousef/Week10#79]
The city where the Yakuza game series takes place. Go do some Karaoke in a Japanese bar.

**Comment by NDunning1** [comment]
I agree japan is a very good choice. It would be really cool to experience a different culture.

**Comment by LiamMackenzie1** [comment]
Good choice
